x-pack/plugins/security_solution/public/management/pages/policy/view/host_isolation_exceptions/host_isolation_exceptions_tab.test.tsx

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { getFoundExceptionListItemSchemaMock } from '../../../../../../../lists/common/schemas/response/found_exception_list_item_schema.mock';
+import { EndpointDocGenerator } from '../../../../../../common/endpoint/generate_data';
+import { PolicyData } from '../../../../../../common/endpoint/types';
+import {
+  AppContextTestRender,
+  createAppRootMockRenderer,
+} from '../../../../../common/mock/endpoint';
+import { getPolicyHostIsolationExceptionsPath } from '../../../../common/routing';
+import { getHostIsolationExceptionItems } from '../../../host_isolation_exceptions/service';
+import { PolicyHostIsolationExceptionsTab } from './host_isolation_exceptions_tab';
+
+jest.mock('../../../host_isolation_exceptions/service');
+jest.mock('../../../../../common/components/user_privileges/endpoint/use_endpoint_privileges');
```

Comment:
I had been trying to remove mocking this "internal" hook to the `useUserPrivileges`. Any change you can mock that instead and not access this one directly?

Reply:
i am not really using this mock. I had to mock it because the http request that this component is doing was failing my tests.